tools/config/__init__.py

```diff
@@ -610,24 +610,62 @@ def _get_mem_specs(self, memories, cmsis_part, exception_text):
                 continue
         raise ConfigException(exception_text)
 
-    @property
-    def rom(self):
-        """Get rom information as a pair of start_addr, size"""
+    def get_region_list(self, region='ROM'):
```

Comment:
This function seems... unnecessary. It's a switch between 2 constants that could be defined at the top of the file.